### DISCUSSION TITLE
Getting Joint Positions of the Human

### DISCUSSION THREAD

**Discussion by hyhalexis** [empriselab/RCareWorld#112] [Help - Programming] [answered]
I saw another post and it seems that there are functions that return the joint positions of the human in the limb repositioning env. Are we able to directly access the joint positions of the human in the dressing env?

**Comment by YoruCathy** [comment, marked as answer]
Nope, to mimic real-world scenarios, we don't directly provide the low-level human joint positions. You might be able to access a human's base position, though.
